cloudformation/auxiliary_templates/custom_balancer/custom_queue_job_balancer.yaml

```diff
@@ -0,0 +1,436 @@
+Description: An AWS Serverless Application Model template describing your function.
+Parameters:
+  FarmId:
+    Description: The Deadline Farm that you are adding a balancer for
```

Comment:
nit: It's the farm ID specifically, so maybe "The Farm ID of the Deadline Farm..."